x-pack/plugins/cloud_security_posture/public/pages/vulnerabilities/vulnerabilities_finding_flyout/vulnerability_finding_flyout.tsx

```diff
@@ -25,15 +27,22 @@ import { VulnerabilityOverviewTab } from './vulnerability_overview_tab';
 import { VulnerabilityJsonTab } from './vulnerability_json_tab';
 import { SeverityStatusBadge } from '../../../components/vulnerability_badges';
 import { VulnerabilityRecord } from '../types';
+import { SEVERITY_STATUS_VULNERABILITY_FLYOUT } from '../test_subjects';
 
 const overviewTabId = 'overview';
 const jsonTabId = 'json';
 
 export const VulnerabilityFindingFlyout = ({
   closeFlyout,
+  onPaginate,
+  totalVulnerabilitiesCount,
+  flyoutIndex,
   vulnerabilityRecord,
 }: {
   closeFlyout: () => void;
+  onPaginate: (pageIndex: number) => void;
+  totalVulnerabilitiesCount: number;
+  flyoutIndex?: number | undefined;
```

Comment:
the `| undefined` here is given by making `flyoutIndex` an optional field
```suggestion
  flyoutIndex?: number;
```